packages/webpack-build-scripts/package.json

```diff
@@ -15,7 +15,6 @@
         "fork-ts-checker-notifier-webpack-plugin": "^3.0.0",
         "fork-ts-checker-webpack-plugin": "^6.1.0",
         "mini-css-extract-plugin": "^1.3.5",
-        "node-sass": "^5.0.0",
```

Comment:
I believe we need to add a `"sass"` dependency here after this change: https://webpack.js.org/loaders/sass-loader/#implementation